frontend/src/metabase/nav/components/AdminNavbar/AdminNavItem.tsx

```diff
@@ -1,15 +1,17 @@
 import React from "react";
-import cx from "classnames";
-import Link from "metabase/core/components/Link";
 import { AdminNavLink } from "./AdminNavItem.styled";
 
-interface AdminNavItem {
+interface AdminNavItemProps {
   name: string;
   path: string;
   currentPath: string;
 }
 
-export const AdminNavItem = ({ name, path, currentPath }: AdminNavItem) => (
+export const AdminNavItem = ({
+  name,
+  path,
+  currentPath,
+}: AdminNavItemProps) => (
```

Comment:
ESLint seems to have trouble when both types and variables are the same names. However, it's a good practice to differentiate between type and component anyway.